includes/reader-revenue/class-stripe-connection.php

```diff
@@ -81,14 +81,31 @@ public static function api_permissions_check( $request ) {
 	 * Get Stripe data blueprint.
 	 */
 	public static function get_default_stripe_data() {
+		$location_code = 'US';
+		$currency      = 'USD';
+
+		// The instance might've used WooCommerce. Read WC's saved settings to
+		// provide more sensible defaults.
+		$wc_country          = get_option( 'woocommerce_default_country', false );
+		$valid_country_codes = wp_list_pluck( newspack_get_countries(), 'value' );
+		if ( in_array( $wc_country, $valid_country_codes ) ) {
```

Comment:
The value coming from `woocommerce_default_country` comes with more specific location code separated by `:`. E.g.: `US:CA` for United States (US) - California and `BR:SP` for Brazil - São Paulo.

We should match the first part of the incoming value.

Reply:
Fixed in 173e5c7d69cd48ca5371378522b22dfaeb093742, and added some tests.